.rustfmt.toml

```diff
@@ -1 +1 @@
-license_template_path = "LICENSE_TEMPLATE"
\ No newline at end of file
+# license_template_path = "LICENSE_TEMPLATE"
```

Comment:
Why did you need this change?

Reply:
`cargo fmt` throws linting error about it, comment it for now.